design/webhook.go

```diff
@@ -0,0 +1,22 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
```

Comment:
isn't the convention to `. import` here?

Reply:
it's. a convention for all goadesign based projects,